lib/carmen/region.rb

```diff
@@ -20,6 +20,18 @@ def name
       Carmen.i18n_backend.translate(path('name'))
     end
 
+    def numeric_code?
+      /[0-9]/ === code[0]
```

Comment:
Even though there aren't any cases currently where subregion codes mix digits and letters, I think I would prefer to see this cheek whether the entire string consists of digits (`/^[0-9]+$/ === code`).

If, for some crazy reason, there comes a time when subregion codes mix letters and digits, checking entirely for digits will make sure that we get consistent behaviour. (For example, what if there were codes 'M32' or '32M'?)

Reply:
Agreed, may I suggest the bulletproof variant though: `/\A\d+\z/` :)